charts/intel-gaudi-resource-driver/README.md

```diff
@@ -0,0 +1,51 @@
+# Dynamic Resource Allocation (DRA) Intel Gaudi Driver Helm Chart
+
+## The chart installs Gaudi resource driver:
+
+- [Gaudi](https://github.com/intel-innersource/containers.orchestrators.kubernetes.intel-resource-drivers-for-kubernetes/doc/gaudi/README.md)
```

Comment:
This is internal repository, nobody from outside Intel will be able to open this link. Same about other places, like two lines below. Needs to be replaced with public GH repository address.

Reply:
https://github.com/intel/helm-charts/pull/58/files#r1708947878

Reply:
yes, I will change the doc for gpu driver in a separate PR not to interfere with this one